packages/eslint-plugin-react-hooks/src/ExhaustiveDeps.js

```diff
@@ -1856,7 +1856,7 @@ function isAncestorNodeOf(a, b) {
 
 function isUseEffectEventIdentifier(node) {
   if (__EXPERIMENTAL__) {
-    return node.type === 'Identifier' && node.name === 'useEffectEvent';
+    return node.type === 'Identifier' && (node.name === 'useEffectEvent' || node.name === 'experimental_useEffectEvent') ;
```

Comment:
I don't think this makes sense. It wouldn't be caught by Rules of Hooks etc anyway. I think what we'd want to do is re-export it internally without the prefix when trying it.